src/main/scala/com/rbmhtechnology/calliope/EventReader.scala

```diff
@@ -29,7 +30,7 @@ trait EventReader[A] {
   * Mixin for applying an arbitrary gap-detection to an [[EventReader]].
   */
 trait GapDetectionEventReader[A] extends EventReader[A] with GapDetection {
```

Comment:
Thanks for the input, I will think about a different solution and create a separate PR for that